deploy/cdk_exec_policy/cdkExecPolicy.yaml

```diff
@@ -74,7 +74,17 @@ Resources:
               - 'quicksight:SetGroupMapping'
               - 'quicksight:RegisterUser'
               - 'quicksight:Describe*'
-            Resource: '*'
+            Resource: 
```

Comment:
This is going to solve the checkov scan problem but technically this is not safe because it allows access to all quicksight resources on the account. But like you said there's no way to limit by prefix? We don't use quicksight at the moment so it's not a massive issue for us. I also hope when we modularize this file we can simply skip this statement altogether. Maybe the data.all team can comment if we can make this more restrictive?

Reply:
I think we can add conditions to only add particular statements if the feature is enabled - i'll work with mourya and see if we can update this PR accordingly